lib/Jsrt/ChakraCommon.h

```diff
@@ -217,7 +217,7 @@ typedef unsigned short uint16_t;
         /// </summary>
         JsErrorInvalidContext,
         /// <summary>
-        ///     Module evaluation is called in wrong context.
+        ///     Invalid Module HostInfoKind provided to JsSetModuleHostInfo or JsGetModuleHostInfo.
```

Comment:
I'd avoid mentioning specific APIs for future-proofing and just say `The Module HostInfoKind provided is invalid.`